ci/scripts/run-test-network-private.sh

```diff
@@ -25,10 +25,10 @@ function stopNetwork() {
 
 # Run Javascript application
 createNetwork
-#print "Initializing Javascript application"
-#pushd ../asset-transfer-private-data/application-javascript
-#npm install
-#print "Executing app.js"
-#node app.js
-#popd
```

Comment:
In #290 the CI is already enabled & passing

https://github.com/hyperledger/fabric-samples/blob/fe04cdcb65fc50d22fcb31b8a045e2ae12ee9e61/ci/scripts/run-test-network-private.sh

Reply:
Correct, a later update I added erroneously removed the the collection config, Dave fixed it for me